tests/test_app.py

```diff
@@ -37,3 +37,41 @@ def test_research_app():
     ]
     result = application_testing(LightningAppTestInt, command_line)
     assert result.exit_code == 0
+
+
+def test_custom_model_via_url_is_available():
+    cwd = os.getcwd()
+    cwd = os.path.join(cwd, "app.py")
+    command_line = [
+        "SD_VARIANT=https://huggingface.co/stabilityai/stable-diffusion-2-1/resolve/main/v2-1_768-ema-pruned.ckpt",
```

Comment:
HF models require providing an access token so the test CI won't be able to download the weights.

Reply:
Check. I will revise it to reflect your opinion.

Reply:
That link is public so access token doesn't need to download the file of link.